v20.2/sql-statements.md

```diff
@@ -165,19 +165,19 @@ Statement | Usage
 [`RESUME JOB`](resume-job.html) | Resume a paused `BACKUP`, `RESTORE`, `IMPORT`, or `CHANGEFEED` job.
 [`SHOW JOBS`](show-jobs.html) | View information on jobs.
 
-## Backup and restore statements (Enterprise)
-
-The following statements require an [enterprise license](enterprise-licensing.html).
-
-{{site.data.alerts.callout_info}}
-For non-enterprise users, see [Back up Data](backup.html) and [Restore Data](restore.html).
-{{site.data.alerts.end}}
+## Backup and restore statements
 
 Statement | Usage
 ----------|------------
 [`BACKUP`](backup.html) | Create disaster recovery backups of databases and tables.
 [`RESTORE`](restore.html) | Restore databases and tables using your backups.
 [`SHOW BACKUP`](show-backup.html) | List the contents of a backup.
+[`CREATE SCHEDULE FOR BACKUP`](create-schedule-for-backup.html) | <span class="version-tag">New in v20.2:</span> Create a schedule for periodic backups.
```

Comment:
So TECHNICALLY in the future, we might support something like `CREATE SCHEDULE FOR REFRESH MATERIALIZED VIEW`

So everything relating to schedules is *generic* and `CREATE SCHEDULE FOR` is the generate create statement. Everything after that is the "statement" or "command" to be run on the schedule, followed by schedule specifics.

Long way of saying, up to you but you may want to preempt that in how you organize the docs.